src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -8,18 +8,28 @@ import {
   getRegionName
 } from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
 
+import { InstantDownloadTranscript } from 'src/shared/components/instant-download';
+
+import { ReactComponent as CloseIcon } from 'static/img/shared/close.svg';
+
+import { Gene } from 'src/content/app/entity-viewer/types/gene';
 import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
 
 import transcriptsListStyles from '../DefaultTranscriptsList.scss';
 import styles from './TranscriptsListItemInfo.scss';
 
 type ItemInfoProps = {
+  gene: Gene;
   transcript: Transcript;
 };
 
 const ItemInfo = (props: ItemInfoProps) => {
+  const [isDownloadShown, setIsDownloadShown] = useState(false);
```

Comment:
How about `isDownloadOpened` instead of `isDownloadShown` since that's what we are using `setIsDownloadShown` to update the state?

Reply:
>  since that's what we are using setIsDownloadShown to update the state

I am not sure I quite understand what you mean, or that the download is a thing that "opens" (I thought what was opening/closing was more like the footer of the info panel; and the download element is its content). What is it about the "shown" that you do not like?

Reply:
Apologies. I got them mixed up. Ignore my suggestion 😃